src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -315,16 +318,21 @@ internal static MemoryInfoResult GetMemoryInfo(SmapsReader smapsReader = null, b
             {
                 extended &= PlatformDetails.RunningOnLinux == false;
 
+                MemoryInfoResult result;
                 using (var process = extended ? Process.GetCurrentProcess() : null)
                 {
                     if (PlatformDetails.RunningOnPosix == false)
-                        return GetMemoryInfoWindows(process, extended);
+                        result = GetMemoryInfoWindows(process, extended);
+                    else if (PlatformDetails.RunningOnMacOsx)
+                        result = GetMemoryInfoMacOs(process, extended);
+                    else result = GetMemoryInfoLinux(smapsReader, extended);
+                }
 
-                    if (PlatformDetails.RunningOnMacOsx)
-                        return GetMemoryInfoMacOs(process, extended);
+                var totalScratchAllocated = GetTotalScratchAllocatedMemory();
+
+                result.AvailableMemory -= new Size(totalScratchAllocated, SizeUnit.Bytes);
```

Comment:
FYI. You can do: `result.AvailableMemory.Add(-totalScratchAllocated, SizeUnit.Bytes)`